launchcontainer/static/js/src/launchcontainer.js

```diff
@@ -51,13 +51,13 @@ function LaunchContainerXBlock(runtime, element) {
         $project = event.target.project.value;
         $token = event.target.token.value;
 
-        var supportEmail = "{{ support_email }}";
+        var supportUrl = "{{ support_url }}";
         clearTimeout(timeoutMessage);
-        if (supportEmail) {
+        if (supportUrl) {
           timeoutMessage = setTimeout(function() {
             $launch_notification.append(
-              '<br /><br />Lab taking a long time to load? Try contacting ' + 
-              '<a href="mailto:' + supportEmail + '">' + supportEmail + '</a>'
+              '<br /><br />Lab taking a long time to load? Contact us for support. ' + 
+              '<a href="supportUrl">' + supportUrl + '</a>'
```

Comment:
Step 2) We still need to show the support email to avoid breaking existing content, but only show it when it's been set. Now it defaults to `None` it will be mostly not set.


```js
if  (supportEmail) {
  $launch_notification.append('Try contacting <a href="mailto:' + supportEmail + '">' + supportEmail + '</a>')
} else {
  $launch_notification.append('<a href="/help">Contact us for support.</a>')
}